File: 3.php
<?php

echo findFirstStringInBracket("Apakah ini akan diambil ? (Jawab)") . PHP_EOL;

function findFirstStringInBracket($str)
{
    $result = "";
    $obracket = strpos($str, "(");
    $cBracket = strpos($str, ")");

    if ( $obracket > 0 && $cBracket > 0 &&  $obracket < $cBracket) {
        $result =  substr($str, ++$obracket, $cBracket - $obracket);
    }

    return $result;
}
?>

File: 3.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	fmt.Println(findFirstStringInBracket("Apakah ini akan diambil ? (Jawab)"))
}

func findFirstStringInBracket(str string) string {
	var result string

	if oBracket, cBracket := strings.Index(str, "("), strings.Index(str, ")"); oBracket > 0 && cBracket > 0 && oBracket < cBracket {
		result = str[oBracket+1 : cBracket]
	}

	return result
}
